app/proguard-rules.pro

```diff
@@ -20,6 +20,7 @@
 # https://github.com/shiftyjelly/pocketcasts-android/issues/1656
 -keep class au.com.shiftyjelly.pocketcasts.core.player.** { *; }
 
+-dontwarn au.com.shiftyjelly.pocketcasts.*.R$style
```

Comment:
We could move it to this section.

https://github.com/Automattic/pocket-casts-android/blob/0afaee9d5f9d686a7efe2fcef8c92766d9efeebb/app/proguard-rules.pro#L164-L174

Reply:
Ah yes. Updated in c6e1ba868236a4f9c9eea6bf68a2029f8f158372.